atom/renderer/atom_sandboxed_renderer_client.cc

```diff
@@ -147,6 +147,7 @@ void AtomSandboxedRendererClient::InitializeBindings(
   b.Set("process", process);
 
   AtomBindings::BindProcess(isolate, &process, metrics_.get());
+  AddRenderBindings(isolate, process.GetHandle());
```

Comment:
Do you have an example of how this might work? I'm not sure how I would inject this into a specific file. Would we want to compile `content-scripts-injector.js` into its own file then inject it with `native_module_loader.CompileAndCall`?

Reply:
@miniak I've made the change you requested regarding the `getRenderProcessPreferences` bindings. lmk if this is what you had in mind.

I'll continue looking into the document start/end events.

Reply:
Okay, I've got the process `document-start` and `document-end` events  working now too. 😄 
0d8f85209bb8fceba505bb2124fc98b941d807bc